content/docs/quickstart/quickstart.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+set -e
+
+# istio settings
+ISTIO_VERSION=1.10.0
+INGRESS_IP=127.0.0.1
+INGRESS_PORT=8082
```

Comment:
This might not always work.

For me on Linux I did not set those variables, and used the installation script from the tutorials
```
curl -o configure-istio.sh https://raw.githubusercontent.com/keptn/examples/release-0.8.3/istio-configuration/configure-istio.sh
```

where we fetch the ingress IP (and do not need a port).

For me on Linux it lead to

```
http://10.XXX.YYY.2.nip.io
```

Reply:
that didn't work for me due to the port 8082. 
is that a bug in Keptn or do I have to fix it on my end?